todoman/model.py

```diff
@@ -567,6 +567,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             with a ``-`` prepended will be used to sort in reverse order.
         :param bool reverse: Reverse the order of the todos after sorting.
         :param int due: Return only todos due within ``due`` hours.
+        :param str priority: Only return todos with priority greater than
```

Comment:
Replace "greater than" with "at least as high as" here (`>` vs `>=`), same in the documentation of `--priority` in `cli.py`.